From 2c0460f79600105373623bf0db2979d22da9dacf Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Wed, 10 Oct 2018 15:30:34 +0800
Subject: [PATCH] ARM64: dts: rockchip: add hwlock node for rk1808

This patch adds Hardware spinlock node support for rk1808 SoC.

Change-Id: I53b3413bfca6a7310a192d029c9fe922fe48964a
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index ad06102ae370..94509264cbd8 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -314,6 +314,12 @@
 		};
 	};
 
+	hwlock: hwspinlock@ff040000 {
+		compatible = "rockchip,hwspinlock";
+		reg = <0 0xff040000 0 0x10000>;
+		#hwlock-cells = <1>;
+	};
+
 	gic: interrupt-controller@ff100000 {
 		compatible = "arm,gic-v3";
 		#interrupt-cells = <3>;
-- 
2.35.3

